core/arch/aarch64/codec.txt

```diff
@@ -257,7 +258,7 @@ x0110111xxxxxxxxxxxxxxxxxxxxxxxx  tbnz   tbz
 ## Unconditional branch (register)
 
 1101011000011111000000xxxxx00000  br     : x5
-1101011000111111000000xxxxx00000  blr    : x5
+1101011000111111000000xxxxx00000  blr    impx30: x5
```

Comment:
just curious, do we have a line for bl, seems we do
https://github.com/DynamoRIO/dynamorio/blob/master/core/arch/aarch64/codec.txt#L214
should we add impx30 for it too?